src/desktop-entry.c

```diff
@@ -23,6 +26,23 @@ struct sfdo {
 static void
 log_handler(enum sfdo_log_level level, const char *fmt, va_list args, void *tag)
 {
+	/*
+	 * libsfdo logging is only provided when LABWC_DEBUG_LIBSFDO is set to
+	 * avoid disproportionately verbose logging by default for one
+	 * particularly sub-system.
+	 */
+	if (!debug_libsfdo) {
+		return;
+	}
```

Comment:
What about this one?
```c
if (!debug_libsfdo && level < SFDO_LOG_LEVEL_ERROR) {
	return;
}
```

Reply:
Assume you mean `level > SFDO_LOG_LEVEL_ERROR` so that we only silence debug+info when `LABWC_DEBUG_LIBSFDO` is unset. (`debug > info > error`)

If so, I like it.

Reply:
Whoops, yes. Didn't recognize that the enum is "the wrong way around" for both wlroots and libsfdo. Guess we could also just do a `if (!debug_libsfdo && level != SFDO_LOG_LEVEL_ERROR) {`, I have no idea what `WLR_SILENT` / `SFDO_LOG_LEVEL_SILENT` even means and why it is above the error loglevel.